aca_weekly_report/report.py

```diff
@@ -583,11 +584,11 @@ def main():
     if not os.path.exists(outdir):
         os.makedirs(outdir)
 
-    stop = opt.stop
+    stop = DateTime(opt.stop)
     if opt.start is None:
-        start = DateTime() - opt.days_back
```

Comment:
The original (DateTime() - opt.days_back) was better. The negative days hack is questionable and specific to just DateTime (not implemented in CxoTime).

Reply:
OK.  If you like that as more explicit that is fine.  I was mostly thinking that start should be a DateTime type as soon as possible instead of a float or integer.  Does that suggest `DateTime(DateTime() - opt.days_back)`?

Reply:
`DateTime() - opt.days_back` is a DateTime already.

Reply:
Oh that's a good point...